demos/virtual_ai_assistant_demo/requirements.txt

```diff
@@ -17,4 +17,5 @@ librosa==0.10.2
 pypdf==4.3.1
 pyyaml==6.0.1
 
-gradio==4.42.0
+gradio==4.43.0
```

Comment:
gradio 4.43 generates errors for me. Let's stick to 4.42.0

Reply:
Gradio 4.42.0 Did not work on my side. Which OS are you using?

Reply:
Windows 11. Are you sure you had 4.42.0 before? I pinned it to that version when you all had problems with the gradio errors. Please verify

Reply:
Yes, I am sure I used 4.42.0 before and I faced the same problem. So When I install 4.43.0and fastapi it works.

Reply:
ok, let me verify again

Reply:
Yes. The problem still exists when using voice input. No issues with 4.42.0